src/lib/utils/date.test.ts

```diff
@@ -342,3 +343,48 @@ test('getDateParts should return null for falsy values', () => {
 test('parseDate should return null for inputs that do not match any date format', () => {
   expect(parseDate('')).toBe(null)
 })
+
+describe('formatEventDateTime', () => {
+  beforeEach(() => {
+    // Check if we're in UTC (CI environment)
+    const isUTC = new Date().getTimezoneOffset() === 0
+    const mockTimezone = isUTC ? 'EST' : 'PST'
+
+    jest.spyOn(Intl, 'DateTimeFormat').mockImplementation(() => ({
+      formatToParts: () => [
+        {
+          type: 'timeZoneName',
+          value: mockTimezone,
+        },
+      ],
+      format: () => '',
+      resolvedOptions: () => ({
+        locale: 'en-US',
+        calendar: 'gregory',
+        numberingSystem: 'latn',
+        timeZone: isUTC ? 'America/New_York' : 'America/Los_Angeles',
+      }),
+      formatRange: () => '',
+      formatRangeToParts: () => [],
+    }))
+  })
+
+  afterEach(() => {
+    jest.restoreAllMocks()
+  })
+
+  it('formats date and time with timezone', () => {
+    const mostRecentDate = {
+      value: 1683216000,
+      endValue: 1683223200,
+    }
+
+    const result = formatEventDateTime(mostRecentDate)
+    const [datePart, timePart] = result.split(' – ')
+    expect(datePart).toMatch(/Thu\. May 4, 2023, \d{1,2}:\d{2} [ap]\.m\./)
+    expect(timePart).toMatch(/\d{1,2}:\d{2} [ap]\.m\. [A-Z]T/)
```

Comment:
I would expect this to expect a specific value, since we're providing a specific value. Like, either it's `9:00 a.m. - 11:00 a.m. PT` or `noon - 3:00 p.m. ET`.

Can we add tests to check that the time zone is being translated correctly as well as the a.m. / p.m. / noon / midnight? This test checks that the format _looks_ correct, not that the times actually _are_ correct.